core/src/main/scala/org/bitcoins/core/protocol/dlc/models/OracleOutcome.scala

```diff
@@ -29,7 +29,7 @@ sealed trait OracleOutcome {
 
   /** The adaptor point used to encrypt the signatures for this corresponding CET. */
   def sigPoint: ECPublicKey = {
-    SigPointComputer(() => computeSigPoint).compute
+    computeSigPoint
   }
```

Comment:
This should be stored in a `val` in the end, as it currently stands this appears to re-compute on each call.